mitmproxy/tools/console/flowview.py

```diff
@@ -139,6 +159,9 @@ def view_tcp_stream(self) -> urwid.Widget:
 
 
         widget_lines = []
+
+        widget_lines.append(self._contentview_status_bar(viewmode.capitalize(), viewmode))
```

Comment:
I have doubts about `viewmode.capitalize()`. Originally there should be `description` of `_get_content_view` (e.g. current contentview's pretty-printed name or error an message if contentview can't be applied). However TCP has several messages as opposed to HTTP (where we have only one message) so it's kind of strange to print errors here because part of messages may parse correctly and part may parse incorrectly.

I propose to leave this stub for now and create an issue in which we'll discuss what's better to be displayed here. One of possible scenarios is to show how much messages (in percentage of total) produced a decode error.

@mhils, what do you think?